concent-vm/Vagrantfile

```diff
@@ -0,0 +1,92 @@
+Vagrant.configure("2") do |config|
+
+    config.trigger.before :up do
+        if ENV['CONCENT_DEPLOYMENT_VERSION'].nil?
+            puts "Please specify `CONCENT_DEPLOYMENT_VERSION` before execute `vagrant up`"
+            abort
+        end
+    end
+
+    config.vm.box      = "debian/stretch64"
+    config.vm.hostname = "concent-development-vm"
+    config.vm.synced_folder(".", "/vagrant", disabled: true)
+
+
+    config.vm.network "private_network", ip: "172.40.2.3"
+
+    blockchain_disk = './disk/blockchain_disk.vdi'
+    config.vm.provider "virtualbox" do |virtualbox|
+        virtualbox.memory = 2048
+        virtualbox.gui    = false
+        if not File.exists?(blockchain_disk)
+            virtualbox.customize [
+                'createhd',
+                '--filename', blockchain_disk,
+                '--variant',  'Fixed',
+                '--size',     30 * 1024, # MB
+            ]
+        end
+        virtualbox.customize [
+            'storageattach',
+            :id,
+            '--storagectl',   'SATA Controller',
+            '--port',         1,
+            '--device',       0,
+            '--type',         'hdd',
+            '--medium',       blockchain_disk,
+            '--hotpluggable', 'on'
+        ]
+    end
+
+    if not File.exists?(blockchain_disk)
+        config.vm.provision "shell",privileged: true, inline: <<-SHELL
+            mkfs.ext4 /dev/sdb
+        SHELL
+    end
+
+    config.vm.provision "shell", run: "always",privileged: true, inline: <<-SHELL
+        directory=/home/vagrant/blockchain
+        if [ ! -d "$directory" ]; then
+            mkdir $directory || exit 1
+        fi
+        mountpoint --quiet $directory || mount /dev/sdb $directory
+    SHELL
+
+    config.trigger.after :halt do
+        info "detach drive"
+        machineId = File.read(".vagrant/machines/default/virtualbox/id")
+        run(
+            "VBoxManage storageattach '#{machineId}' " +
+            "--storagectl 'SATA Controller' "          +
+            "--port       1 "                          +
+            "--device     0 "                          +
+            "--type       hdd "                        +
+            "--medium     none"
+        )
+    end
+
+    config.trigger.before :destroy do
+        # The vagrant destroy virtual machine with attached disk.
+        # We don't want to destroy disk which contain blockchain, so before destroy we attached another disk with parameter none
+        info "detach drive"
```

Comment:
Please reword: 
``` ruby
        # `vagrant destroy` normally destroys any virtual disks attached to the machine.
        # Our disk contains Ethereum blockchain which can take a lot of time to redownload so we detach it to prevent its destruction.
        info "Detaching (but not removing) virtual disk #{blockchain_disk}."
```

Reply:
Done.